src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -3040,6 +3040,7 @@ PyMODINIT_FUNC PyInit_lpython_module_)" + fn_name + R"((void) {
         std::string out;
         std::string indent(4, ' ');
         switch (x.m_intrinsic_id) {
+            SET_INTRINSIC_NAME(ObjectType, "type");
```

Comment:
Does the integration test added in this PR work with the `c` backend. If it works, then enable the `c` backend as well. If not, then I think we need not update the `c` backend here.

Reply:
It works. I have added the `c` back-end. :+1:

Reply:
Perfect.